presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,40 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+History
+-------
+
+The CLI keeps a history of your previously used commands. You can access your
+history by scrolling or searching. Use the up and down arrows to scroll and
+:kbd:`Control+S` and :kbd:`Control+R` to search. To execute a query again,
+press **Enter**.
+
+By default, you can locate the Presto history file in ``~/.presto_history``.
+Use the ``PRESTO_HISTORY_FILE`` environment variable to change the default.
+
+Troubleshooting
+---------------
+
+If something goes wrong, an error message displays:
```

Comment:
"displays" sounds a bit strange to me. How about "is shown"?

Reply:
I'm not in love with "is shown" either. Per the Google standards we could do a slightly more passive sentence that sounds more natural when spoken "If something goes wrong, you'll see an error message:" or "... error message like the following:" - preferences?

Reply:
Maybe avoiding future tense use:

If some goes wrong, you see an error message: